packages/data-bite/README.md

```diff
@@ -0,0 +1,11 @@
+# `data-bite`
```

Comment:
Please fill this out - reference other packages to see

Reply:
Done